TileFormats/GeometryData/README.md

```diff
@@ -0,0 +1,108 @@
+
+| Field name | Data type | Description |
+| --- | --- | --- |
+| `magic` | 4-byte ANSI string | `"geom"`. This can be used to identify the arraybuffer as a Geometry tile. |
+| `version` | `uint32` | The version of the Geometry Data format. It is currently `1`. |
+| `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of two fields: `Feature Table` and  `Batch Table`.
+
+Code for reading the header can be found in
+[Geometry3DTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/vector-tiles/Source/Scene/Geometry3DTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `geom` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `geom` Feature Table JSON schema is defined in [geom.featureTable.schema.json](../../schema/geom.featureTable.schema.json).
+
+### Semantics
+
+Per-feature semantics specific to a feature type are prefixed with the name of the feature type. e.g. `BOXES` for boxes, `CYLINDERS` for cylinders and `ELLIPSOIDS` for ellipsoids.
+
+At least one global `LENGTH` semantic must be defined. 
+If `BOXES_LENGTH` in not defined, or zero, no boxes will be rendered.
+If `CYLINDERS_LENGTH` in not defined, or zero, no cylinders will be rendered.
+If `ELLIPSOIDS_LENGTH` in not defined, or zero, no ellipsoids will be rendered.
+If `SPHERES_LENGTH` in not defined, or zero, no spheres will be rendered.
+Multiple feature types may be defined in a single Geometry tile using multiple `LENGTH` semantics, and in that case, all specified feature types will be rendered.
+
+If a semantic has a dependency on another semantic, that semantic must be defined as well.
+
+#### Geometry Semantics
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `BOXES` | `float32[]` | The boxes in the tile. The length of the array will be `19 * BOXES_LENGTH`. The first three elements of each box are the x, y, and z dimensions. The following 16 elements are the 4x4 model matrix in column-major order. | :white_check_mark: Yes, when the global `BOXES_LENGTH` is greater than zero. |
```

Comment:
Let's be super explicit - are the dimensions the full length or half length?

Reply:
Do we even need this since it could just be the translation part of the 4x4 matrix?  Just all boxes are unit boxes transformed by the matrix?

Reply:
I'm OK with just a model matrix, that's how the implementation works.

Reply:
Should it mention anything about scaling the unit geometry? The only one I thought worth mentioning was the cylinder for which axes scaled the radius and which one was the length.

Reply:
+1 for the cylinder axes.  I don't think anything else is needed for the box.